### PR TITLE
Implement tmt run --remove to remove workdir

### PR DESCRIPTION
**Pull request by psss**
Support an easy way to cleanup the workdir after testing is
finished. Works nice when selected steps are enabled as well.
Basic test and documentation included.

### CODE REVIEW

**Comment by lukaszachy** [comment]
How should `--last` behave in case of `--remove`?

File: tmt/cli.py [hunk 1]
```diff
@@ -158,6 +158,9 @@ def main(context, root, **kwargs):
     '-i', '--id', 'id_', help='Run id (name or directory path).', metavar="ID")
 @click.option(
     '-l', '--last', help='Execute the last run once again.', is_flag=True)
+@click.option(
+    '-r', '--remove', is_flag=True,
```
[the remark this answers is not in the document]
Comment (psss) [reply tmt/cli.py:162]: Sure, no problem. Added.

**Comment by psss** [comment]
Thanks for the review!

> How should `--last` behave in case of `--remove`?

Using `--last` works fine when running selected steps. For example:

    tmt run --remove --until execute
    tmt run --last --after execute

The removal happens only when `finish` step is enabled. But then the last run is gone. Same with all steps enabled. As for now I don't know how to make this better. We would have to keep history of last runs. But that would not solve the problem either and could be confusing.

**Comment by lukaszachy** [comment]
Thank you for adding `--rm`

**Comment by psss** [comment]
> Thank you for adding `--rm`

Sure! And thanks to tests I found out that "just" adding `--rm` completely broke the feature ;-) Updated code already in testing...

**Pull request merged**